Lib/ufo2ft/featureWriters/quantizedMarkFeatureWriter.py

```diff
@@ -0,0 +1,41 @@
+from ufo2ft.featureWriters import MarkFeatureWriter
+import math
+from collections import OrderedDict
+
+def quantize(number, degree):
+    return degree * math.floor(number / degree)
```

Comment:
instead of math.floor, I suggest you use `otRound(number / degree) * degree` for quantizing the values, so it will round halfway up towards +Infinity.
https://github.com/fonttools/fonttools/blob/aae11e45c29f4d07770574ce8577e6e61bc2b15c/Lib/fontTools/misc/roundTools.py#L21-L42